torchbenchmark/models/hf_Bert_large/metadata.yaml

```diff
@@ -0,0 +1,10 @@
+devices:
+  NVIDIA A100-SXM4-40GB:
+    eval_batch_size: 4
+eval_benchmark: false
+eval_deterministic: false
+eval_nograd: true
+not_implemented:
+- jit: true
+train_benchmark: false
```

Comment:
what does this mean?

Reply:
This is config for CI; I just copied these from hf_Bert/metadata.yaml (and reduced batch size). Not sure exactly what train_benchmark does, (@xuzhao9 do you know? I did some grepping but still couldn't tell what it was doing)